identity-composer-component.js

```diff
@@ -144,6 +144,19 @@ function Ctrl($q, brCredentialLibraryService, brMediaQueryService) {
       });
   };
 
+  self.scaleWidth = function(scale) {
+    // remove the vw unit
+    var width = self.credentialWidth.select.slice(0,-2);
+    var scaledWidth = (scale * width).toFixed(2);
+    return scaledWidth.toString() + 'vw';
+  };
```

Comment:
This seems kinda hacky to me ... but we can move forward with it and address later. Please mark with a "TODO: could be improved, seems hacky" so we can note it for later :)